source/infrastructure/free-radius/cloudwatch-errors.html.md.erb

```diff
@@ -0,0 +1,34 @@
+---
+title: Common FreeRADIUS Cloudwatch errors
```

Comment:
Could this be Cloudwatch Alarms rather than errors, since that's the term Cloudwatch uses I think?

Reply:
Right, I've mistaken these for other things :+1: